python/ray/data/_internal/execution/interfaces.py

```diff
@@ -191,22 +191,6 @@ class ExecutionOptions:
     """Common options for execution.
 
     Some options may not be supported on all executors (e.g., resource limits).
-
```

Comment:
I was able to fix this via adding the nosummary template: https://github.com/ray-project/ray/pull/34228/files#diff-b3a1d55836e0b2035d287e569a716fa02296221b15eee9babd6c2ea0f62a2ec7R13

Reply:
cool, thanks

Reply:
@pcmoritz you can undo this particular diff now (fixed in master).

Reply:
Thanks, will do!